### PR TITLE
#162171009 Eslint configuration

### PR DESCRIPTION
**Pull request by edogbosunny**
#### What does this PR do?
It configures eslint to the project
#### Description of Task to be completed?
- Install eslint library modules in the project
- write eslint configurations to a new .eslintrc file
- Write a script to fix syntax errors
#### How should this be manually tested?
npm run lint-fix
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[Finishes #162171009]
#### Screenshots (if appropriate)
Nil
#### Questions:
Nil

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,93 +1,15 @@
...
-            error: {}
-        }
-    });
-});
-
-// finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, function() {
-    console.log("Listening on port " + server.address().port);
+app.listen(PORT, () => {
+  console.log(`connected on port ${PORT}`);
```
[the remark this answers is not in the document]
Comment (thebere) [reply index.js]: Install and use logger for debug or info purposes https://www.npmjs.com/package/logger

[the remark this answers is not in the document]
Comment (edogbosunny) [reply index.js]: Thank you for your feedback, i would implement this as soon as possible

File: index.js [hunk 2]
```diff
@@ -11,5 +13,5 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
 app.listen(PORT, () => {
-  console.log(`connected on port ${PORT}`);
-});
\ No newline at end of file
+  nodeLogger.info(`connected on port ${PORT}`);
```
Comment (thebere) [inline index.js]: Use `logger` instead of `nodeLogger` which is too verbose

Comment (edogbosunny) [reply index.js]: Thank you for the feed back i would rename to logger variable

**Pull request merged**